sympy/printing/latex.py

```diff
@@ -34,18 +34,29 @@
 tex_greek_dictionary = {
     'Alpha': 'A',
     'Beta': 'B',
+    'Gamma': r'\Gamma',
+    'Delta': r'\Delta',
     'Epsilon': 'E',
     'Zeta': 'Z',
     'Eta': 'H',
```

Comment:
Is `\Eta` really supported by standard LaTeX? I can't find it (neither `\Alpha`, etc.) in the comprehensive list of symbols: http://ctan.mines-albi.fr/info/symbols/comprehensive/SYMLIST